README.md

```diff
@@ -27,6 +27,47 @@ make all
 
 If you have access to our internal servers, you can run `make nonfree-init` to fetch additional resources we cannot make publically accessible. Note that these are *not required* to use anything provided in this repository.
 
+## Linux image
```

Comment:
Since we are building doc in the branch `aottaviano/update-readme` would it make sense to remove this md snippet and move it in the aforementioned branch? Mostly for @paulsc96

Reply:
As this make target has been added in this PR, is it possible to keep it there and merge conflicts later?

Reply:
Sure

Reply:
Done